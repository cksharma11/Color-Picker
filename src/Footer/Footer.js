import React from "react";

const Footer = () => {
  return <div className="div-footer">Footer Text</div>;
};

export default Footer;
