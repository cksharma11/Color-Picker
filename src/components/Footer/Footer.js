import React from "react";

const Footer = () => {
  return (
    <div className="div-footer">With great color, comes great design.</div>
  );
};

export default Footer;
