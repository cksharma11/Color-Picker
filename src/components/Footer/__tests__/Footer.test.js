import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Footer from "../Footer";

Enzyme.configure({ adapter: new Adapter() });

const colorList = [
  { colorName: "red", colorCode: "#FF0000" },
  { colorName: "green", colorCode: "#008000" }
];
describe("Footer", () => {
  it("should render properly", () => {
    const wrapper = shallow(<Footer colors={colorList} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
