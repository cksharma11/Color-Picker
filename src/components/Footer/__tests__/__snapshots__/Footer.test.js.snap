// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render properly 1`] = `
"<div className=\\"div-footer\\">
  With great color, comes great design.
</div>"
`;
